docs/quickstart.md

```diff
@@ -0,0 +1,125 @@
+# Quickstart
+
+The following section provides the info regarding how to prepare and use
+`ralph-cli`.  After reading it, you'll be able to perform a successful scan of
+your iDRAC/iLO host, and save its result to Ralph (at this moment, it will be
+limited only to MAC addresses, but that is going to change soon!).
+
+## Getting ralph-cli binary
```

Comment:
btw i'll consider naming this section 'Installation' (i know that downloading binary is not a installation per se, but it could be more familiar to the user when he'll see headers of the docs than 'Getting ralph-cli binary')